styles.css

```diff
@@ -36,7 +36,8 @@ ul.nolist {
 	display: block;
 	width: 100%;
 	height: 100vh;
-	background: url('/images/background.jpg') center top no-repeat;
+	background: url('/assets/backgrounds/rotate.php') no-repeat center center fixed;
+	background-size: cover;
```

Comment:
The call to `rotate.php` is what kicks off the random background picking. Setting it to centered horizontally/vertically, fixed position, and then using background-size: cover ensures it will always fill up the browser window and center.